src/journey-maps/angular/components/popup/popup.scss

```diff
@@ -0,0 +1,2 @@
+// Popup style needs to be global
```

Comment:
Nitpick: Why not simply declare global popup styles here and disable view encapsulation for this component?